apps/crn-frontend/src/network/teams/TeamOutput.tsx

```diff
@@ -116,11 +100,7 @@ const TeamOutput: React.FC<TeamOutputProps> = ({ teamId }) => {
             setErrors(clearAjvErrorForPath(errors, instancePath))
           }
           onSave={(output) =>
-            createResearchOutput({
-              ...defaultOutput,
-              ...output,
-              addedDate: new Date().toISOString(),
-            }).catch((error) => {
+            createResearchOutput(output).catch((error) => {
```

Comment:
Should this be a try...catch rather than a promise chain?

Reply:
![image](https://user-images.githubusercontent.com/1936253/165563612-0d474d50-723f-486b-8d5c-f1d4aa24ed27.png)

This is what it would look like if I converted it into async. You have to do the weird return await in order to catch the error from the parent promise. You also have to return a Promise.Resolve as the caught error is now handled which is ugly too. Tempted to leave this as is.